### PR TITLE
Parametrizes tests for smoothn3

### PR DESCRIPTION
**Pull request by dpshelio**

### CODE REVIEW

**Comment by dpshelio** [comment]
Now it works!! Phewww.. sorry for the previous noise. I started to do this as a review for #92 but it was closed and merged into #95 and I needed to some rebasing.

File: kaska/tests/test_smoothn.py [hunk 1]
```diff
@@ -80,68 +51,49 @@ def test_sd_weights():
...
-
-    target_rms = np.array([[0.04707677, 0.33686247, 0.66873105],
-                           [1.37314063, 1.03404982, 1.36475258],
-                           [2.57311525, 2.46561701, 1.80441455]])
-
-    residual_rms_assessment(s, d, target_maximum_residuals, target_rms)
+@pytest.mark.parmetrize("weights, target_max_resid, target_rms",[['masked',
+                                                                  np.array([[0.19063412, 1.63927078, 3.22016034],
+                                                                            [2.86830946, 3.3060494 , 5.26835424],
+                                                                            [4.74814422, 6.95324161, 4.21698349]])
```
Comment (timspainUCL) [inline kaska/tests/test_smoothn.py]: This line needs a comma at the end of it (I think!)

File: kaska/tests/test_smoothn.py [hunk 2]
```diff
@@ -80,68 +51,49 @@ def test_sd_weights():
...
+                                                                  np.array([[0.18979927, 1.63548774, 3.21602835],
+                                                                            [2.87126492, 3.30231171, 5.26648978],
+                                                                            [4.69832211, 6.88804564, 4.19266901]]),
+                                                                  np.array([[0.04338182, 0.31158228, 0.61451283],
+                                                                            [1.39932148, 1.00197386, 1.306045  ],
+                                                                            [3.00532792, 2.80878405, 1.85278847]])],
+                                                                 ['cauchy',
+                                                                  np.array([[0.20149125, 1.72772566, 3.42323339],
+                                                                            [2.78516476, 3.45869778, 5.50720599],
+                                                                            [4.28855901, 6.58983133, 4.37961953]])
```
Comment (timspainUCL) [inline kaska/tests/test_smoothn.py]: And here

Comment (dpshelio) [reply kaska/tests/test_smoothn.py]: Good catch!!

**Pull request merged**